### DISCUSSION TITLE
Condition is not working

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#9243] [Archive]
Hello! I use this condition to hide interface in table `research_works`:

````
{
   "_and": [
      {
         "user_created": {
            "active_research": {
               "id": {
                  "_null": true
               }
            }
         }
      }
   ]
}
````
but nothing changes

`active_research` is FK to  `research_works`

I use Directus 9.0.0.rc-98

Have you any ideas?

UPD:
It seems that relative filters like `o2m` and `m2m` are not working in interface conditions...

**Comment by rijkvanzanten** [comment]
> It seems that relative filters like o2m and m2m are not working in interface conditions...

This is correct, also see https://github.com/directus/directus/discussions/7123
